repo-assets/app.js

```diff
@@ -0,0 +1,88 @@
+$(() => {
+
+    console.log("DEBUG js working");
+    var firebaseConfig = {
+        apiKey: "AIzaSyAU8mVJA8grfzu394GaYpsnLO5_BbTvwuA",
+        authDomain: "train-scheduler-3c591.firebaseapp.com",
+        databaseURL: "https://train-scheduler-3c591.firebaseio.com",
+        projectId: "train-scheduler-3c591",
+        storageBucket: "train-scheduler-3c591.appspot.com",
+        messagingSenderId: "984798529891",
+        appId: "1:984798529891:web:bf8e0ec3cf73957522eb56"
+    };
+      
+      firebase.initializeApp(firebaseConfig);
+      var db = firebase.database();
+
+    //   var newTrainName;
+    //   var newDestination;
+    //   var newTrainTime;
+    //   var newFrequency; 
+
+    // function nextTrain(newTrainTime) { 
+    //     var timeDiff = moment().diff(moment(newTrainTime, "minutes"));
+    //     var timeRemainder = timeDiff % newFrequency;
+    //     var minutesAway = newFrequency - timeRemainder;
+    //     var arrivingTrain = moment().add(minutesAway, "minutes");
+    //     arrivingTrain = moment(arrivingTrain).format("hh:mm A");
+
+    
+    // };
```

Comment:
Remove commented code before you make a PR! Typically in the wild, commented code and console.logs() would not be allowed in the core repo, so you always want to remove them.